src/compiler/factory/nodeFactory.ts

```diff
@@ -7081,32 +7161,51 @@ function updateWithOriginal<T extends Node>(updated: Mutable<T>, original: T): T
 
 function getDefaultTagNameForKind(kind: JSDocTag["kind"]): string {
     switch (kind) {
-        case SyntaxKind.JSDocTypeTag: return "type";
-        case SyntaxKind.JSDocReturnTag: return "returns";
-        case SyntaxKind.JSDocThisTag: return "this";
-        case SyntaxKind.JSDocEnumTag: return "enum";
-        case SyntaxKind.JSDocAuthorTag: return "author";
-        case SyntaxKind.JSDocClassTag: return "class";
-        case SyntaxKind.JSDocPublicTag: return "public";
-        case SyntaxKind.JSDocPrivateTag: return "private";
-        case SyntaxKind.JSDocProtectedTag: return "protected";
-        case SyntaxKind.JSDocReadonlyTag: return "readonly";
-        case SyntaxKind.JSDocOverrideTag: return "override";
-        case SyntaxKind.JSDocTemplateTag: return "template";
-        case SyntaxKind.JSDocTypedefTag: return "typedef";
-        case SyntaxKind.JSDocParameterTag: return "param";
-        case SyntaxKind.JSDocPropertyTag: return "prop";
-        case SyntaxKind.JSDocCallbackTag: return "callback";
-        case SyntaxKind.JSDocOverloadTag: return "overload";
-        case SyntaxKind.JSDocAugmentsTag: return "augments";
-        case SyntaxKind.JSDocImplementsTag: return "implements";
+        case SyntaxKind.JSDocTypeTag:
+            return "type";
+        case SyntaxKind.JSDocReturnTag:
+            return "returns";
+        case SyntaxKind.JSDocThisTag:
+            return "this";

```

Comment:
Here's a big case where I do wonder if we need to have an option that allows this or something.